drivers/rackspace/rackspace.go

```diff
@@ -76,12 +76,12 @@ func GetCreateFlags() []cli.Flag {
 		cli.StringFlag{
 			Name:  "rackspace-image-id",
 			Usage: "Rackspace image ID. Default: Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)",
-			Value: "",
 		},
 		cli.StringFlag{
-			Name:  "rackspace-flavor-id",
-			Usage: "Rackspace flavor ID. Default: General Purpose 1GB",
-			Value: "general1-1",
+			Name:   "rackspace-flavor-id",
+			Usage:  "Rackspace flavor ID. Default: General Purpose 1GB",
+			Value:  "general1-1",
+			EnvVar: "RACKSPACE_FLAVOR_ID",
```

Comment:
The others env var names have the OpenStack prefix `OS_`. I think we should use the same rule here (`OS_FLAVOR_ID`) for naming consistency.